modules/microshift-automation-example-bootc-systems.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-automation-example-bootc-systems_{context}"]
+= Automating the integration process with systemd for bootc systems
+
+[IMPORTANT]
+====
+You must include the entire `auto-recovery` process for bootc systems that use `systemd` in the container file.
+====
+
+As a use case, consider the following example situation in which you want to automate the `auto-recovery` process for bootc systems that use systemd.
+
+.Procedure
+
+* Create your own container file by running the following commands, for example:
++
+[source,terminal]
+----
+$ COPY ./auto-rec/10-auto-recovery.conf /usr/lib/systemd/system/microshift.service.d/10-auto-recovery.conf
+----
++
+[source,terminal]
+----
+$ COPY ./auto-rec/microshift-auto-recovery.service /usr/lib/systemd/system/
+----
++
+[source,terminal]
+----
+$ COPY ./auto-rec/microshift-auto-recovery /usr/bin/
```

Comment:
```suggestion
$ cp ./auto-rec/10-auto-recovery.conf /usr/lib/systemd/system/microshift.service.d/10-auto-recovery.conf
----
+
[source,terminal]
----
$ cp ./auto-rec/microshift-auto-recovery.service /usr/lib/systemd/system/
----
+
[source,terminal]
----
$ cp ./auto-rec/microshift-auto-recovery /usr/bin/
```
"COPY" is not a valid unix command, I assume it's meant to be "cp" to copy the file

Reply:
@pmtk WDYT about this comment? I see we're creating a container here. Could you clarify?

Reply:
Yes, these are contents of the Containerfile that user would use to prepare bootc image. They should be together (in a single snippet).
```
COPY ./auto-rec/10-auto-recovery.conf /usr/lib/systemd/system/microshift.service.d/10-auto-recovery.conf
COPY ./auto-rec/microshift-auto-recovery.service /usr/lib/systemd/system/
COPY ./auto-rec/microshift-auto-recovery /usr/bin/
```

Reply:
Thanks for that explanation. so rather than stating "Create your own container file by running the following commands, for example:" we should be stating "Add the following lines to the containerfile that you use to prepare the bootc image"?

Reply:
updates are made